### DISCUSSION TITLE
Where to enter custom analytics tracking code

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1842] [Help] [answered]
Hi
How and where do I enter custom analytics JS tracking code (publytics), which has no plugin yet.
Thx.

**Comment by carstenfeller** [comment, marked as answer]
Hi treelistener

go to "Tools & Plugins", chose "Custom HTML"

![publii-custom-html](https://github.com/user-attachments/assets/c758030e-848a-4ae6-8d6f-a440dfddded9)

then use the "Head" or "Body" option as recommended to place the code snippet.

![publii-custom-html-1](https://github.com/user-attachments/assets/48304130-840a-4183-8dc8-b577a9cbfdd5)

Sync your website.

Works with Plausible, GA4 or Matomo so should work with your analytics provider too.

C.F.

**Reply by treeandcoffee** [reply]
Oh yes, thank you very much. I didnt realized that I could post the tracking code there.

**Reply by candidexmedia** [reply]
That's exactly it! Also works for counter.dev, Swetrix, etc.

Just make sure to check where the code is meant to go. Sometimes, scripts have to go at the end.

**Reply by treeandcoffee** [reply]
That works great with Publytics and the brandnew Litlyx.
